collectors/cgroups.plugin/multi_metadata.yaml

```diff
@@ -1025,7 +1035,9 @@ modules:
         - ovirt
         - container
     overview:
+      <<: *overview
       data_collection:
+        <<: *data_collection
         metrics_description: "Monitor oVirt for performance, resource usage, and health status."
```

Comment:
This collector doesn't monitor oVirt for performance, resource usage, and health status